salt/haproxy/haproxy.cfg.jinja

```diff
@@ -0,0 +1,28 @@
+global
+        log /dev/log    local0
+        log /dev/log    local1 notice
+        daemon
+
+defaults
+        log     global
+        mode    tcp
+        option  tcplog
+        option  dontlognull
+        timeout connect 5000
+        timeout client 50000
+        timeout server 50000
+
+listen kubernetes-master
+        bind 127.0.0.1:6444
```

Comment:
Ideally, we should listen on the standard kube-apiserver port here so any pre-existing configs don't need a port update once this lands, and the default port is more obvious for people when thinking it through later.

On the masters, we should probably also bind to 0.0.0.0, so end users will enter through the same LB.